plugins/arElasticSearchPlugin/config/mapping.yml

```diff
@@ -630,5 +630,5 @@ mapping:
             search_analyzer: standard
             term_vector: with_positions_offsets
       # Not nested date fields for sorting
-      start_date_sort: { type: date, include_in_all: false }
-      end_date_sort: { type: date, include_in_all: false }
+      start_date_sort: { type: date, copy_to: _all }
+      end_date_sort: { type: date, copy_to: _all }
```

Comment:
Same for a lot of fields, there is a lot. If the field had `include_in_all: false` it should not have `copy_to: _all`. And, looking at the condition, we only care about `text` types, so we only need to add `copy_to: _all` to those when needed.